modules/apps/headless/headless-asset-library/headless-asset-library-impl/src/main/java/com/liferay/headless/asset/library/internal/resource/v1_0/AssetLibraryResourceImpl.java

```diff
@@ -527,6 +534,26 @@ private UnicodeProperties _toUnicodeProperties(Settings settings) {
 		).build();
 	}
 
+	private void _updateDLSizeLimitConfiguration(
+			long groupId, AssetLibrary assetLibrary)
+		throws Exception {
+
+		Map<String, Long> mimeTypeSizeLimits = new LinkedHashMap<>();
+
+		Settings settings = assetLibrary.getSettings();
+
+		MimeTypeLimit[] mimeTypeLimits = settings.getMimeTypeLimits();
+
+		for (MimeTypeLimit mimeTypeLimit : mimeTypeLimits) {
+			mimeTypeSizeLimits.put(
+				mimeTypeLimit.getMimeType(),
+				GetterUtil.getLong(mimeTypeLimit.getMaximumSize()));
+		}
+
+		_dlSizeLimitConfigurationProvider.updateGroupSizeLimit(
```

Comment:
Should `fileMaxSize` and `maxSizeToCopy` be set to `0L`?

Reply:
It takes the value of the company or system configuration, doesn't it?

Reply:
see com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java:146.

We are doing the same (in spaces we do not have the "fileMaxSize" configuration yet)